src/datacatalog_tag_manager/datacatalog_facade.py

```diff
@@ -30,6 +30,21 @@ def create_or_update_tag(self, parent_entry_name, tag):
             logging.info('%sCreated: %s', self.__NESTED_LOG_PREFIX, created_tag.name)
             return created_tag
 
+    def delete_tag(self, parent_entry_name, tag):
+        entry_tags = self.__datacatalog.list_tags(parent=parent_entry_name)
+
+        try:
+            persisted_tag = next(
+                entry_tag for entry_tag in entry_tags
+                if entry_tag.template == tag.template and entry_tag.column == tag.column)
```

Comment:
What happens here if the tag is not attached to a column?

Reply:
both values are empty, and it finds the right tag

Reply:
Cool, thanks!